modules/table/src/main/java/org/apache/ignite/internal/table/distributed/storage/InternalTableImpl.java

```diff
@@ -248,6 +254,10 @@ public void name(String newName) {
         this.tableName = newName;
     }
 
+    private String txCoordinatorId() {
```

Comment:
I'd rather add coordinatorId to InternalTransaction and thus remove given method from InternalTableImpl. What I mean here is that txManager should propagate coordinatorId to InternalTransaction on tx.begin().